Makefile

```diff
@@ -15,3 +15,6 @@ pylint-onnx:
 		nncf/experimental/onnx				\
 		nncf/quantization					\
 		tests/onnx examples/experimental/onnx
+
+test-install-onnx:
+	pytest tests/cross_fw/install/test_install.py --backend onnx
```

Comment:
Should we specify a folder (tests/cross_fw/install/) but not a test_install.py file?

Reply:
Done